io/plugins/eu.esdihumboldt.hale.io.gml/src/eu/esdihumboldt/hale/io/gml/writer/internal/StreamGmlWriter.java

```diff
@@ -1597,6 +1606,11 @@ private void writeProperties(Group parent, Collection<? extends ChildDefinition<
 				&& ((GroupPropertyDefinition) parent.getDefinition())
 						.getConstraint(ChoiceFlag.class).isEnabled();
 
+		// DEBUGGING
+		if (parent.getDefinition().getIdentifier().equals("identifier")) {
```

Comment:
Remove?

Do you know that you can also use conditional breakpoints in Java?

Reply:
No....